dev/NavigationView/NavigationView_InteractionTests/CommonTests.cs

```diff
@@ -1598,5 +1598,21 @@ public void VerifyExpandCollpaseFunctionality()
                 }
             }
         }
+
+        [TestMethod]
+        public void VerifyNoCrashWhenNavViewInContentDialog()
+        {
+            var testScenarios = RegressionTestScenario.BuildLeftNavRegressionTestScenarios();
+            foreach (var testScenario in testScenarios)
+            {
+                using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", testScenario.TestPageName }))
+                {
+                    Log.Comment("Open a ContentDialog with a NavView inside.");
+                    Button openContentDialogButton = new Button(FindElement.ById("ContentDialogNavViewButton"));
+                    openContentDialogButton.Click();
+                    Wait.ForIdle();
```

Comment:
>ForIdle [](start = 25, length = 7)

Did you verify that this fails without your product changes? I'm concerned there would be a timing issue and this could pass even though it would crash a second later.

Reply:
Yeah, I did verify that test fails without the fix. However you do have a point, I updated the test to close the ContentDialog.